src/Extensions/TimedScopes/Internal/ActivityObserversIntializer.cs

```diff
@@ -128,5 +143,10 @@ private void OnActivityStopped(Activity activity, object payload)
 				stopHandler.OnStop(activity, payload);
 			}
 		}
+
+		private void OnException(string eventName, object payload)
+		{
+			m_logger.LogError(Tag.Create(), payload as Exception, "Exception diagnostic event '{0}'", eventName);
```

Comment:
Do we lose any granularity here by treating it as Exception, rather than reconstructing the original type?
I'd assume not since each sub-type should implement it's own ToString but just want to be sure we aren't losing anything.

Reply:
we won't because log method itself accepts Exception, the problem might be if payload is not Exception but some custom object that contains exception field inside, but I'm not sure how to handle it